test(tools): Fix saunafs command tests for Windows

rolysr opened the pull request:
Previous changes for tests for the saunafs command on for Windows were having a flaky behavior for changes from commit bd88d8b. Due to that, the changes from that commit were reverted in commit 8f2b674. This commit reintroduces the corresponding changes so that this needed fixes for the saunafs command tests on Windows are applied again.

antuan96314 commented:
> Previous changes for tests for the saunafs command on for Windows were having a flaky behavior for changes from commit [bd88d8b](https://github.com/leil-io/saunafs/commit/bd88d8b1da3224022a17a1aee2475184e04a8fb7). Due to that, the changes from that commit were reverted in commit [8f2b674](https://github.com/leil-io/saunafs/commit/8f2b674a643539375e1e734c91061f0eecb7141d). This commit reintroduces the corresponding changes so that this needed fixes for the saunafs command tests on Windows are applied again.

Please review commit message.

rolysr committed:
test(tools): Fix saunafs command tests for Windows

Previous changes for tests for the saunafs command on for Windows were
having a flaky behavior for changes from commit bd88d8b. Due to that,
the changes from that commit were reverted in commit 8f2b674.
This commit reintroduces the corresponding changes so that a first set
of the stable tests can be added of now. The idea is to add tests by
groups of them, wich will be faster approach than trying to directly
integrating all tests at once.

Signed-off-by: rolysr <rolysr@leil.io>
